src/api/actions/ActionsAPI.js

```diff
@@ -43,31 +43,30 @@ class ActionsAPI extends EventEmitter {
         this._allActions[actionDefinition.key] = actionDefinition;
     }
 
-    get(objectPath, viewContext, options) {
+    get(objectPath, viewProvider, options) {
```

Comment:
We don't need options here. [See previous comments about this](https://github.com/nasa/openmct/pull/3325#discussion_r494534416). The view context should provide everything needed for `appliesTo` to decide whether or not to include the action.